### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by 0xC0ncord**
Some various fixes including changes needed to fix new AVCs with systemd 255.

With this set of changes we also implement the proposed fix in https://github.com/SELinuxProject/refpolicy/issues/732.

### CODE REVIEW

File: policy/modules/system/init.if [hunk 1]
```diff
@@ -775,6 +775,26 @@ interface(`init_exec',`
...
+##	entrypoint for the calling domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <rolecap/>
+#
+interface(`init_exec_no_trans',`
```
Comment (pebenito) [inline policy/modules/system/init.if:793]: Dupe of interface above it.

File: policy/modules/kernel/kernel.te [hunk 2]
```diff
@@ -337,6 +337,7 @@ mls_process_set_level(kernel_t)
 selinux_getattr_fs(kernel_t)
 selinux_load_policy(kernel_t)
 
+
```
Comment (pebenito) [inline policy/modules/kernel/kernel.te:340]: Unnecessary extra line.

**Review by pebenito — CHANGES_REQUESTED** [review]

File: policy/modules/kernel/kernel.te [hunk 3]
```diff
@@ -390,10 +390,14 @@ ifdef(`init_systemd',`
 	')
 
 	optional_policy(`
+		dev_getattr_generic_usb_dev(kernel_t)
```
Comment (pebenito) [inline policy/modules/kernel/kernel.te:393]: The `getattr` is also provided by `dev_delete_generic_usb_dev(kernel_t)`; see `delete_chr_file_perms`.

File: policy/modules/kernel/kernel.te (hunk 2 above)
Comment (pebenito) [reply policy/modules/kernel/kernel.te:340]: Still showing an extra newline.

File: policy/modules/system/init.if (hunk 1 above)
Comment (pebenito) [reply policy/modules/system/init.if:793]: Still showing the unnecessary interface.

File: policy/modules/kernel/kernel.te (hunk 2 above)
Comment (0xC0ncord) [reply policy/modules/kernel/kernel.te]: Sorry -- these snuck back in while juggling this PR between multiple workstations.

**Review by pebenito — APPROVED** [review]

**Pull request merged**